microprofile/microprofile-opentracing-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/opentracing/v13/interfaces/NotTraced.java

```diff
@@ -0,0 +1,31 @@
+package org.wildfly.swarm.ts.microprofile.opentracing.v13.interfaces;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import org.wildfly.swarm.ts.microprofile.opentracing.v13.TracedService;
+
+@Path("/interface/nottraced")
+public class NotTraced {
```

Comment:
How is this _not traced_? All JAX-RS endpoints are traced by default, and I don't see any configuration to switch it off for this class.

Reply:
It seems this is just an endpoint that's used by the client that isn't traced. Why not point the not-traced client to `/` (class `Resource`)? It should be the same, no?